starter-project-web-vue2/components/abel/AbelButton.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <button @click="$emit('toggle-task')">{{ text }}</button>
```

Comment:
I would recommend using a vuetify button. and using classes to make it look how you want it to. check more about it [here](https://vuetifyjs.com/en/components/buttons/
) but if you needed this exact component, It's ok